src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,19 @@
+package duke.command;
+
+import java.util.HashMap;
+
+import duke.TaskList;
+import duke.Ui;
+
+// Print out a bye message and do necessary clean up
```

Comment:
Use Javadoc comment to describe the purpose of a class

Reply:
Thanks for the advice. This has been fixed in the latest commit.